com.civicrm.hrjobroles/js/src/job-roles/filters/option-group-filter.js

```diff
@@ -0,0 +1,19 @@
+define([
+  'job-roles/filters/filters'
+], function(filters) {
+  'use strict';
+
+  filters.filter('getActiveGroup', ['$log', function($log) {
+    $log.debug('Filter: getActiveGroup');
+
+    return function(groupData, param) {
+      var filteredGroupData = [];
+      angular.forEach(groupData, function(group, key) {
```

Comment:
Also, please always use lodash instead of angular for utility functions like `forEach` on objects
```js
define([
  'common/lodash'
], function (_) {

  _.each(groupDate, function (group) {

  });
});